integrations/datadog/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,73 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: host
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"datadogHost"'
+          identifier: .id | tostring
+          title: .name
+          properties:
+            up: .up
+            hostName: .host_name
+            description: .description
+            metrics: .metrics
+            sources: .sources
+  - kind: monitor
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"datadogMonitor"'
+          identifier: .id | tostring
+          title: .name
+          properties:
+            tags: .tags
+            monitorType: .type
+            overallState: .overall_state
+            thresholds: .thresholds
+            priority: .priority
+            createdBy: .creator.name
+            createdAt: .created
+            updatedAt: .modified
+  - kind: serviceCatalog
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"datadogServiceCatalog"'
+          identifier: .attributes.schema."dd-service"
+          title: .attributes.schema."dd-service"
+          properties:
+            contacts: .attributes.schema.contacts
+            languages: .attributes.schema.languages
+            description: .attributes.schema.description
+            tags: .attributes.schema.tags
+            type: .attributes.schema.type
+            code_repositories: .attributes.schema.links | map(.url)
```

Comment:
Since the UI nicely handles links, I have renamed the field instead to **Service Links**.

<img width="1189" alt="Screenshot 2024-05-10 at 18 33 54" src="https://github.com/port-labs/ocean/assets/8656262/0d0d3c4b-a7f4-4518-8a2e-9397f12ec1c1">